pkg/streamingpromql/functions/common.go

```diff
@@ -5,8 +5,10 @@ package functions
 import (
 	"github.com/grafana/mimir/pkg/streamingpromql/pooling"
 	"github.com/grafana/mimir/pkg/streamingpromql/types"
+	"github.com/prometheus/prometheus/promql"
 )
 
+// SeriesMetadataFunction is a function to operate on the metadata of a single series.
```

Comment:
This isn't quite right - `SeriesMetadataFunction` operates over many series, not a single series.